index.html

```diff
@@ -11,7 +11,7 @@
    <header>
     <nav class="navbar">
         <div class="logo-div">
-            <p class="logo">My Logo</p>
+            <p class="logo"><a href="#first-section">My Logo</a></p>
```

Comment:
- [ OPTIONAL ] Why do you remove the `p` tag from here so you use only the `a` tag since it is not a paragraph?